packages/facebook-oauth/package.js

```diff
@@ -0,0 +1,18 @@
+Package.describe({
+  summary: "Facebook OAuth flow",
+  version: "1.3.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use('underscore', 'server');
+  api.use('random', 'client');
+  api.use('service-configuration', 'client');
```

Comment:
I believe `service-configuration` needs to be `['server', `client']` here otherwise if you don't have `service-configuration` in your `.meteor/packages` it will fail when you try to login after the configuration:

0. Click **Configure Facebook Login**
 
 ![image](https://cloud.githubusercontent.com/assets/841294/19358794/f3a2267c-917f-11e6-97dd-d62010b1435e.png)

0. Fill in then **Save Configuration**

 ![image](https://cloud.githubusercontent.com/assets/841294/19358849/33951ec4-9180-11e6-8b5e-2a608260d356.png)

0. It appears to save, and button changes to login
 
 ![image](https://cloud.githubusercontent.com/assets/841294/19358948/ae4d7558-9180-11e6-911b-cc5d0ffa557f.png)

0. But when you click it, you get Internal Server Error

 ![image](https://cloud.githubusercontent.com/assets/841294/19359152/810e0a02-9181-11e6-8f08-e0c1c7c9cfc5.png)

I tested with `server` and it worked properly.